django_keycloak/__init__.py

```diff
@@ -1 +1,3 @@
+from .token import Token
```

Comment:
Is there a reason for importing Token here? Would also require import update in managers.py

Reply:
The import here ables you to use:
```python
from django_keycloak import Token
```
Consider you are providing an "easier and public" API. Might be useful if someone decides to build a custom integration and needs to interact with the `Token` class.